js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -419,17 +424,59 @@ const AssetGraphExplorerWithData = ({
       shortcutFilter={(e) => e.altKey && e.code === 'KeyE'}
     >
       {expandedGroups.length === 0 ? (
-        <Button
-          title="Expand all groups"
-          icon={<Icon name="unfold_more" />}
-          onClick={() => setExpandedGroups(allGroups)}
-        />
+        <Tooltip
+          content={
+            <>
+              Expand all groups{' '}
+              <Caption
+                style={{
+                  width: 16,
+                  borderRadius: 4,
+                  padding: '2px 4px',
+                  color: colorAccentWhite(),
+                  background: colorAccentGray(),
+                  marginLeft: 4,
+                }}
+              >
+                ⌥E
+              </Caption>
+            </>
+          }
+        >
+          <Button
+            title="Expand all groups"
+            icon={<Icon name="unfold_more" />}
+            onClick={() => setExpandedGroups(allGroups)}
+            style={{background: colorBackgroundDefault()}}
+          />
+        </Tooltip>
       ) : (
-        <Button
-          title="Collapse all groups"
-          icon={<Icon name="unfold_less" />}
-          onClick={() => setExpandedGroups([])}
-        />
+        <Tooltip
+          content={
+            <>
+              Collapse all groups{' '}
+              <Caption
+                style={{
+                  width: 16,
+                  borderRadius: 4,
+                  padding: '2px 4px',
+                  color: colorAccentWhite(),
+                  background: colorAccentGray(),
+                  marginLeft: 4,
+                }}
+              >
```

Comment:
This style is repeated in a few places -- it would make sense to factor it out into a `styled.div` to.